integrationtest/org/daisy/dotify/formatter/test/ExternalReferenceTest.java

```diff
@@ -0,0 +1,111 @@
+package org.daisy.dotify.formatter.test;
+
+import org.daisy.dotify.api.engine.LayoutEngineException;
+import org.daisy.dotify.api.writer.PagedMediaWriterConfigurationException;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import java.io.IOException;
+
+/**
+ * Testing the external reference object interaction with framework.
+ * In this case an external reference tag will be handled, sent through and
+ * added to rows in the output.
+ */
+public class ExternalReferenceTest extends AbstractFormatterEngineTest {
+
+    @Ignore("Not implemented yet, only supporting references on block level")
+    @Test
+    public void testExternalReference1() throws
```

Comment:
Please use a descriptive name or add a comment that describes what you test
Please do this at all tests